.travis.yml

```diff
@@ -15,6 +15,7 @@ install:
 - pip install --upgrade pip
 - pip --version
 - pip install -r pip-freeze.txt --upgrade
+- pip install setuptools==33.1.1  # Recent version breaks packaging.requirements, this is a temp fix: https://github.com/pypa/setuptools/issues/942
```

Comment:
Have you seen this fail on Travis? https://github.com/pypa/setuptools/issues/942#issuecomment-274817619 Seems to suggest we should be ok with latest pip here.